packages/dds/tree/src/simple-tree/api/typesUnsafe.ts

```diff
@@ -83,6 +142,14 @@ export type TreeFieldFromImplicitFieldUnsafe<TSchema extends Unenforced<Implicit
 			? TreeNodeFromImplicitAllowedTypesUnsafe<TSchema>
 			: unknown;
 
+/**
+ * {@link Unenforced} version of {@link AllowedTypes}.
+ * @remarks
```

Comment:
Not necessarily for this PR, but should we add similar notices to the existing `Unsafe` types here as well?

Reply:
Its redundant with the system tag: I'd prefer a more general/automated solution to convey that to intelisense users.